packages/content-management/table_list_view_table/src/components/user_avatar_tip.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { UserAvatarTip as UserAvatarTipComponent } from '@kbn/user-profile-components';
+import { useQuery } from '@tanstack/react-query';
+import { useServices } from '../services';
+
+export function UserAvatarTip(props: { uid: string }) {
+  const { getUserProfile } = useServices();
+  const query = useQuery(
+    ['user-profile', props.uid],
+    async () => {
+      return getUserProfile(props.uid);
+    },
+    { staleTime: Infinity }
+  );
```

Comment:
Because we are only using the `uid` as our query key here, it looks like the custom avatars don't get refetched until after a hard refresh:

https://github.com/elastic/kibana/assets/8698078/e376f7b0-7e71-441d-a266-f5858c6db790

I don't consider this to be a blocker by any means, because I don't think it's critical that this gets updated right away necessarily - especially if this would impact performance. But figured it was worth noting.